server/services/licenceSearchService.js

```diff
@@ -15,14 +15,14 @@ module.exports = function createLicenceSearchService(
   nomisClientBuilder,
   prisonerSearchApi
 ) {
-  const bookingIdForExistingLicence = async (bookingId) => {
-    const licence = await licenceClient.getLicence(bookingId)
+  const bookingIdForLicenceIncludingSoftDeleted = async (bookingId) => {
+    const licence = await licenceClient.getLicenceIncludingSoftDeleted(bookingId)
```

Comment:
this licence search was only being used by the admin search so switched the `getLicence` client call to a new `getLicenceIncludingSoftDeleted` client call which searched the `licences` table instead of the view which excludes soft deleted licences.